qcengine/programs/nwchem/harvester.py

```diff
@@ -599,7 +600,7 @@ def harvest_outfile_pass(outtext):
             # print (mobj.group(2)) #error reason
             psivar['NWCHEM ERROR CODE'] = mobj.group(1)
             # TODO process errors into error var
-
+ 
```

Comment:
be sure to run `make format` as the PR box describes to get rid of little things like this. (ping if that doesn't make sense.)